prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    const fullName = 'Tyler Robinson';
+    console.log('fullName:', fullName);
+    var numberOfPets = '0';
+    console.log('numberOfPets:', numberOfPets);
+    var likesVegetables = 'True';
+    console.log('likesVegetables:', likesVegetables);
+    var nothing = 'null';
```

Comment:
Hi Tyler, be sure to leave out the quotes from numbers, booleans, and `null` so that the browser doesn't interpret them as strings.